udata_hydra/config_default.toml

```diff
@@ -3,6 +3,7 @@ LOG_LEVEL = "DEBUG"
 DATABASE_URL = "postgres://postgres:postgres@localhost:5432/postgres"
 # csv as database table
 DATABASE_URL_CSV = "postgres://postgres:postgres@localhost:5434/postgres"
+DATABASE_SCHEMA = "public"
```

Comment:
It's configured to `public` because it is the default PostgreSQL schema, therefore there is no need to create it.
In case we want to use another one, it should be created and configured at the infra level (and then `DATABASE_SCHEMA` config key should be set accordingly).